Restore old MLL fit (#320)
emdgroup/baybe: CHANGELOG.md

  are not passed on to the GPyTorch kernels
- Positive-valued kernel attributes are now correctly handled by validators
  and hypothesis strategies

> - Replaced `fit_gpytorch_mll` with old `fit_gpytorch_mll_torch` call until finetuning
>   is achieved

Reviewer: ```suggestion
- Reverted `fit_gpytorch_mll` call back to old `fit_gpytorch_mll_torch` call until
  finetuning is achieved
```
